src/types.ts

```diff
@@ -1,10 +1,15 @@
+import { BN } from "@coral-xyz/anchor";
 import {
   CreatePositionEvent,
+  PoolStructure,
   RemovePositionEvent,
+  Tick,
 } from "@invariant-labs/sdk-eclipse/lib/market";
+import { PublicKey } from "@solana/web3.js";
 
 export interface IActive {
   event: CreatePositionEvent;
+  secondsPerLiquidityInside: BN;
```

Comment:
Naming could be slight more accurate. I suggest to name it `previousSnapSecondsPerLiquidityInside` or maybe you can come up with something shorter and more accurate than now